### PR TITLE
Implemented Search form functionality

### PR DESCRIPTION
**Pull request by mabrasheva**

### CODE REVIEW

**Comment by ZheniKoleva** [comment]
LGTM

**Review by ZheniKoleva — APPROVED** [review]

File: static/css/styles.css [hunk 1]
```diff
@@ -252,4 +252,103 @@ button:hover {
     align-items: flex-start;
     align-content: flex-end;
     justify-content: center;
-}
\ No newline at end of file
+}
+
+.background {
```
Comment (ZheniKoleva) [inline static/css/styles.css]: The code from line 257 to the end can be deleted because it duplicates  the code from line 88 to 183

Comment (mabrasheva) [reply static/css/styles.css]: Removed duplicate styles.

**Review by IvayloAnev — COMMENTED** [review]
LGFM

**Pull request merged**